cpp/tests/io/parquet_test.cpp

```diff
@@ -3200,4 +3206,25 @@ TEST_F(ParquetWriterTest, RowGroupSizeInvalid)
                cudf::logic_error);
 }
 
+TYPED_TEST(ParquetWriterSchemaTest, FieldID)
+{
+  column_wrapper<TypeParam> col;
+  std::vector<std::unique_ptr<column>> cols;
+  cols.push_back(col.release());
+  auto expected = std::make_unique<table>(std::move(cols));
+
+  cudf_io::table_input_metadata expected_metadata(*expected);
+  auto constexpr gold = 825;
+  expected_metadata.column_metadata[0].set_parquet_field_id(gold);
+
+  auto filepath = temp_env->get_temp_filepath("FieldID.parquet");
+  cudf_io::parquet_writer_options out_opts =
+    cudf_io::parquet_writer_options::builder(cudf_io::sink_info{filepath}, expected->view())
+      .metadata(&expected_metadata);
+
+  auto got_metadata = out_opts.get_metadata();
```

Comment:
I suppose this test isn't needed anymore?

Reply:
Removed